src/api/cvc4cpp.h

```diff
@@ -2270,15 +2270,6 @@ class CVC4_PUBLIC Solver
   Sort declareDatatype(const std::string& symbol,
                        const std::vector<DatatypeConstructorDecl>& ctors) const;
 
-  /**
-   * Declare 0-arity function symbol.
-   * SMT-LIB: ( declare-fun <symbol> ( ) <sort> )
```

Comment:
Maybe change the comment for `declareSort` to say that it also corresponds to `( declare-fun <symbol> ( ) <sort> )`?

Reply:
Good point, will do.

Reply:
Oh wait, it already does.